source/declarators.tex

```diff
@@ -2388,7 +2388,7 @@
 \indextext{initialization|(}
 
 \pnum
-A declarator can specify an initial value for the
+An initializer can specify an initial value for the
```

Comment:
My understanding here is "can" means "can be used", but obviously it is not clear enough.